nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+const mealsWithReviews = meals.map((meal) => {
+  meal.reviews = [];
+  reviews.forEach((review) => {
+    if (meal.id === review.mealId) {
+      meal.reviews.push(review);
+    }
+  });
+  return meal;
+});
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+app.get("/meals", async (request, response) => {
+  response.send(mealsWithReviews);
```

Comment:
I suggest adding` response.statusCode = 200;` along with `response.send(message);` would be good practice as a beginner as we use more status codes like 400, 500, etc. in the future.